pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -112,19 +139,88 @@ func TestDefaultConfigMap(t *testing.T) {
 				BinaryData: nil,
 			},
 		},
+		{
+			name: "Test new Config Map that would come from managed namespace",
+			args: args{
+				operatorConfig: &operatorv1.Console{
+					TypeMeta:   metav1.TypeMeta{},
+					ObjectMeta: metav1.ObjectMeta{},
+					Spec:       operatorv1.ConsoleSpec{},
+					Status:     operatorv1.ConsoleStatus{},
+				},
+				consoleConfig: &configv1.Console{
+					Spec:   configv1.ConsoleSpec{},
+					Status: configv1.ConsoleStatus{},
+				},
+				managedConfig: &corev1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "ConfigMap",
+						APIVersion: "v1",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "console-config",
+						Namespace: "openshift-config-managed",
+					},
+					Data:       map[string]string{configKey: exampleManagedConfigMapData},
+					BinaryData: nil,
+				},
+				infrastructureConfig: &configv1.Infrastructure{
+					Status: configv1.InfrastructureStatus{
+						APIServerURL: mockAPIServer,
+					},
+				},
+				rt: &routev1.Route{
+					TypeMeta:   metav1.TypeMeta{},
+					ObjectMeta: metav1.ObjectMeta{},
+					Spec: routev1.RouteSpec{
+						Host:              host,
+						Path:              "",
+						To:                routev1.RouteTargetReference{},
+						AlternateBackends: nil,
+						Port:              nil,
+						TLS:               nil,
+						WildcardPolicy:    "",
+					},
+					Status: routev1.RouteStatus{},
+				},
+			},
+			want: &corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{},
```

Comment:
Same here, is it necessary to add all of the boilerplate to the `configmap`, or can we trim it down to only the fields in use?